routing_common/bicycle_model.cpp

```diff
@@ -172,6 +172,9 @@ VehicleModel::SurfaceInitList const kBicycleSurface = {
   {{"psurface", "paved_bad"}, {0.8, 0.8}},
   {{"psurface", "unpaved_good"}, {1.0, 1.0}},
   {{"psurface", "unpaved_bad"}, {0.3, 0.3}},
+  {{"_mtb_rating", "intermediate"}, {0.3, 0.3}}, // similar to unpaved_bad
```

Comment:
I think there will be a lot of cases with `psurface-unpaved_bad` and `_mtb_rating-*` at the same time, so the factors will multiply (which is unwanted)

I think the most simple solution is to ensure there is a `psurface-unpaved_bad` when mtb rating is present (even if there are no surface & smoothness tags).
This way all mtb trails will be considered as "slow" and normally the router will avoid to route all the cyclists there